ui/src/app/modules/invoice/components/invoice-detail/invoice-detail.component.ts

```diff
@@ -133,6 +133,7 @@ export class InvoiceDetailComponent implements OnInit {
 
     public duplicate() {
         this.invoice = this.invoice.copy();
+        this.save();
```

Comment:
Better to keep a transient object and add a CSS animation to inform the copy action has succeed